supertokens_python/recipe/dashboard/recipe.py

```diff
@@ -247,7 +248,7 @@ def reset():
         DashboardRecipe.__instance = None
 
     def return_api_id_if_can_handle_request(
-        self, path: NormalisedURLPath, method: str
+        self, path: NormalisedURLPath, method: str, user_context: Dict[str, Any]
```

Comment:
missing changes for this function - see node PR

Reply:
Fixed.